BT-Tracking/Interface/Current Data/CurrentDataVM.swift

```diff
@@ -85,30 +137,43 @@ final class CurrentDataVM {
                 try? realm.write {
                     self.currentData?.update(with: result)
                 }
-
-                DispatchQueue.main.async {
-                    AppSettings.currentDataLastFetchDate = Date()
-
-                    self.sections = self.sections(from: self.currentData)
-                    self.updateFooter()
-                    self.observableErrors.onNext(nil)
-                }
             } else if let error = error {
                 self.observableErrors.onNext(error)
             }
+            dispatchGroup.leave()
         }
     }
 
-    private func updateFooter() {
-        if let lastFetchedDate = AppSettings.currentDataLastFetchDate {
-            footer = L10n.currentDataFooter(dateFormatter.string(from: lastFetchedDate))
-        }
+    func fetchAppCurrentData(in dispatchGroup: DispatchGroup) {
+        dispatchGroup.enter()
+
+        // swiftlint:disable:next force_unwrapping
+        let url = URL(string: "DownloadMetrics", relativeTo: RemoteValues.serverConfiguration.appCurentDataURL)!
+        AF.request(url)
+            .validate(statusCode: 200..<300)
+            .responseDecodable(of: AppCurrentData.self, decoder: jsonDecoder) { response in
+                #if DEBUG
+                debugPrint(response)
+                #endif
```

Comment:
Mohl by `debugPrint` mit v sobe `#if DEBUG`, aby nebylo nutne ho vzdy podminkovat?

Reply:
Ono to je od Almofire